tests/publiccloud/download_repos.pm

```diff
@@ -80,7 +80,7 @@ sub run {
         }
         # Failsafe: Fail if there are no test repositories, otherwise we have the wrong template link
         my $count             = scalar @repos;
-        my $check_empty_repos = get_var('QAM_PUBLICCLOUD_IGNORE_EMPTY_REPO', 0) == 0;
+        my $check_empty_repos = get_var('PUBLIC_CLOUD_SKIP_DOWNLOAD', 0) == 0;
```

Comment:
I don't see how we come from `QAM_PUBLICCLOUD_IGNORE_EMPTY_REPO` to `PUBLIC_CLOUD_SKIP_DOWNLOAD`. Perhaps you meant `PUBLIC_CLOUD_IGNORE_EMPTY_REPO`?